deploy/kubernetes/setup-sumologic.yaml.tmpl

```diff
@@ -269,7 +269,7 @@ spec:
           value: $SUMOLOGIC_ACCESSID
         - name: SUMOLOGIC_ACCESSKEY
           value: $SUMOLOGIC_ACCESSKEY
-        # - name: SUMOLOGIC_BASE_URL
-        #  value: $SUMOLOGIC_BASE_URL
+        - name: SUMOLOGIC_BASE_URL
+          value: $SUMOLOGIC_BASE_URL
```

Comment:
this doesn't look right. We need to remove `--set sumologic.endpoint="\$SUMOLOGIC_BASE_URL"` from `build.sh`